### DISCUSSION TITLE
Start menu Windows 10 - German language files

### DISCUSSION THREAD

**Discussion by Lyhtande** [valinet/ExplorerPatcher#2146] [Q&A]
Hallo,
can someone provide the necessary files and instructions to make the start menu appear in German?
Thanks in advance!

**Comment by pyrates999** [comment]
What does this have to do with EP?

**Reply by Lyhtande** [reply]
Well, when I set EP to use the Windows 10 start menu, it is displayed in English language. When I switch to the Windows 11 start menu, it is displayed in German language.

**Comment by pyrates999** [comment]
Thank you.

Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by Lyhtande** [reply]
Explorer Patcher Version: 22621.2283.57.2

<a href="https://imgur.com/da2kDVO"><img src="https://i.imgur.com/da2kDVO.png" title="source: imgur.com" /></a>

<a href="https://imgur.com/PlR74Uj"><img src="https://i.imgur.com/PlR74Uj.png" title="source: imgur.com" /></a>

**Comment by pyrates999** [comment]
Ok, did you install the german language version of windows 11?  Or did you install the english version and then added the german language files to it?

**Reply by Lyhtande** [reply]
It's a German language Windows version, with just German languge installed.

**Reply by pyrates999** [reply]
Ok, thank you.

**Reply by Diego-JRH** [reply]
I have the same request but for spanish language. Everything but the start menu is in spanish.

**Comment by Amrsatrio** [comment]
This should be fixed in the latest prerelease.

**Comment by Lyhtande** [comment]
Thank you so much!

**Comment by Diego-JRH** [comment]
It is fixed!, thanks so much.
